examples/debug/example.py

```diff
@@ -20,12 +20,6 @@ class DoubleProcessor(BaseProcessor):
     Example processor that will double the value passed in
 
     """
```

Comment:
All the rest was useful for debugging and developing, but isn't needed at all. Now just outputs the doubled value directly.